Sources/LeafKit/Exports.swift

```diff
@@ -5,6 +5,7 @@ extension UInt8 {
     static let quote: UInt8 = 0x22
     static let octothorpe: UInt8 = 0x23
     static let leftParenthesis: UInt8 = 0x28
```

Comment:
We should convert the rest of these to the `.utf8.first!` syntax. So much better.